wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceManager.java

```diff
@@ -895,13 +895,27 @@ private WorkspaceImpl doCreateWorkspace(WorkspaceConfig config,
     }
 
     private WorkspaceImpl getByKey(String key) throws NotFoundException, ServerException {
+
+        int lastColonIndex = key.indexOf(":");
```

Comment:
Should we fix doc for WorkspaceManager#getWorkspace https://github.com/eclipse/che/pull/4073/files#diff-349ffaa889025cd6acd2d5b512c2e687L183 ?